packages/sandbox/src/components/Preview.tsx

```diff
@@ -78,7 +78,7 @@ export function Preview() {
   }, [accountId, debouncedFiles]);
 
   return (
-    <div className={s.wrapper}>
+    <div className={`${s.wrapper}`}>
```

Comment:
@andy-haynes Curious about this syntax change. This should have been sufficient: `className={s.wrapper}`?